### ISSUE TITLE
Latin x and cyrillic he (U+0425/U+0445) variants similar to new cyrl-zhe=cursive

### ISSUE THREAD

**Issue by pv4** [be5invis/Iosevka#1997]
@be5invis @jmcwilliams403 

Is it possible please to create variants (nearly like on the following picture), similar to new cyrillic zhe variants?

![image](https://github.com/be5invis/Iosevka/assets/2502094/8704de26-daa7-4265-a734-2fc5b48f0b02)
![image](https://github.com/be5invis/Iosevka/assets/2502094/8019b3f4-085d-4e19-991f-d3f53c66172a)

This variant would be visually simplier than the current `x=cursive` and I personally would prefer it (currently I use standard/upright x variant in my italic builds).

In many fonts and in russian hand-writing cyrl-he/x/cyrl-zhe have simialr writing for legs:

1. Iosevka (straight)
![image](https://github.com/be5invis/Iosevka/assets/2502094/0586f1bf-41e0-47d1-be3f-215813a28254)
![image](https://github.com/be5invis/Iosevka/assets/2502094/e4e98423-fec4-4949-8f4a-7b93f3e59be7)

2. russian hand-writing
![image](https://github.com/be5invis/Iosevka/assets/2502094/53bd815e-441e-4d37-aefa-dd93e8a27163)

**Comment by jmcwilliams403** [comment]
The cursive zhe variants were added for `ss12` and `ss15`.
IBM Plex Mono, the source for `ss15`, uses the existing cursive form of `x`/he, while Ubuntu Mono, the source for `ss12`, does not use any special italic form for it.

I have no authority on whether this variant is worth adding.
All I can say is to please provide examples of other monospace/programming fonts that do this.

**Comment by pv4** [comment]
Since what I suggest is some sort of a "cursive" style, not a "simple" one, it's hard to find an existing monospace/programming font of the style in question. At least, I wasn't able to.

But I found some cursive fonts with the style (see below).

Before the images, I have to say that the style I'm asking for is something between the current "normal" and "cursive" styles. It's more cursive than the "normal" one, and it's more normal (and horizontally symmetric) than the current "cursive" style. Why I'm asking it now is because I think the existing cyrl-zhe=cursive legs can be reused somehow, and I think it would be fine to have cyrl-zhe/cyrl-he/x variants that look similar.

And now, the images, all are from https://www.fontspace.com/

https://www.fontspace.com/valley-forge-font-f102566
https://www.fontspace.com/auntha-clear-trial-font-f103009
https://www.fontspace.com/delagio-script-font-f99379
https://www.fontspace.com/safir-script-font-f24141
https://www.fontspace.com/cursif-font-f3640
https://www.fontspace.com/gingerale-font-f43955
https://www.fontspace.com/little-days-font-f10836
https://www.fontspace.com/wrexham-script-font-f5880
https://www.fontspace.com/santos-dumont-font-f11077
https://www.fontspace.com/vtc-nuetattooscript-font-f9341
https://www.fontspace.com/pottery-crafting-font-f46396
https://www.fontspace.com/the-dreamer-font-f11517
https://www.fontspace.com/diamond-dust-font-f19696
https://www.fontspace.com/ink-in-the-meat-font-f11238
https://www.fontspace.com/bridga-font-f68541
https://www.fontspace.com/mardian-font-f13944
https://www.fontspace.com/beauty-mountains-font-f30686
https://www.fontspace.com/inborn-font-f70932
https://www.fontspace.com/christmas-time-star-font-f70379
https://www.fontspace.com/blonde-font-f70195
https://www.fontspace.com/snow-puppets-font-f70176
https://www.fontspace.com/dattermatter-bold-persoinal-use-font-f85390

![image](https://github.com/be5invis/Iosevka/assets/2502094/bdd8680c-8742-44e6-a6b4-e7dfa699515d)

Basically, it's like the current x=cursive but less cursive, more symmetric and with a simplier look.

**Comment by pv4** [comment]
@jmcwilliams403 thanks for the reply and for the explanation!

**Comment by Logo121** [comment]
I do recall a "turn c-c"-like cyrillic ha appearing in some serif fonts, mirroring that of zhe. However for those the hooks on zhe curl more than what's implemented here (the more common shape for monospace fonts), see #1762 for those examples.

Here are some less fancy font samples for ha:
![image](https://github.com/be5invis/Iosevka/assets/21302803/578f5a6e-70e7-4399-9dc7-5f8f1cc1ef99) (Treats zhe a bit differently)
![image](https://github.com/be5invis/Iosevka/assets/21302803/7cb98d36-0e1f-4e0d-af9e-a8f3d83eecbc)
![image](https://github.com/be5invis/Iosevka/assets/21302803/45ad6e23-bde5-45e3-b993-1dec0f787e03)
![image](https://github.com/be5invis/Iosevka/assets/21302803/5dd6b3d8-6c7c-41e7-88e5-a1e90cee624b)

I don't think these are enough justification however, since this is an exclusively serif font thing. The suggested variant looks quite unnatural, at least not really better than the existing cursive.  
Just imo tho.

**Comment by jmcwilliams403** [comment]
As of 549910d , this variant exists:
![image](https://github.com/be5invis/Iosevka/assets/37010132/90472325-662e-4d8c-ac8f-b7cc3f882010)
Is this good enough in your opinion, @pv4 ?

**Comment by pv4** [comment]
> Is this good enough in your opinion, @pv4 ?

@jmcwilliams403 yeah, that's fine! Thank you!

**Comment by pv4** [comment]
@be5invis I think this issue can be closed as fixed (unless you're planning to do something else on it).
